packages/editor/src/components/post-publish-panel/postpublish.js

```diff
@@ -153,8 +160,18 @@ export default function PostPublishPanelPostpublish( {
 							variant="primary"
 							href={ link }
 							__next40pxDefaultSize
+							icon={ external }
+							iconPosition="right"
+							target="_blank"
+							rel="noopener noreferrer"
```

Comment:
```suggestion
							target="_blank"
```

Nit: There is an effort to stop adding `noreferrer` and `noopener`. See  https://github.com/WordPress/gutenberg/issues/26914https://github.com/WordPress/gutenberg/issues/26914